assets/styles/app.scss

```diff
@@ -3,4 +3,44 @@
 body {
     background-color: $ground-color;
     color: $map-color;
+    margin: 0;
 }
+
+#mapid { 
+    height: 50vh; 
+    //border-radius: 45%;
+    margin-top: 50px;
```

Comment:
margin should not be more than 1em. looks like you're dealing with layout more than spacing: use flexbox ;)